implementation/src/main/java/io/smallrye/metrics/elementdesc/RawMemberInfo.java

```diff
@@ -111,4 +111,9 @@ public boolean equals(Object obj) {
         return other.getDeclaringClassName().equals(this.getDeclaringClassName()) &&
                 other.getName().equals(this.getName());
     }
+
+    @Override
+    public int hashCode() {
+        return this.getDeclaringClassName().hashCode() + this.getName().hashCode();
```

Comment:
I think simply adding multiple hashes is not a mathematically proper way to compute a higher-level hash. How about using the convenient JDK method
```
Objects.hash(declaringClassName, name);
```
See https://www.sitepoint.com/how-to-implement-javas-hashcode-correctly/

Reply:
yes that's better, thanks